examples/with-sequelize/src/Home.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import logo from './react.svg';
+import './Home.css';
+
+class Home extends React.Component {
+  state = {
+    users: null,
+  };
+
+  async componentDidMount() {
```

Comment:
This isn’t a good thing to promote. Refactor to just use promises. Make sure to include a polyfill for isomorphic fetch.